baybe/utils/chemistry.py

```diff
@@ -149,10 +149,14 @@ def smiles_to_fingerprint_features(
     )
     name = f"{encoding.name}_"
     prefix = prefix + "_" if prefix else ""
-    col_names = [
-        prefix + name + f.split("fingerprint")[1]
+    no_descriptor_names = all(
+        "fingerprint" in f for f in fingerprint_encoder.get_feature_names_out()
```

Comment:
let's avoid the unnecessary second call to `get_feature_names_out` by storing the sequence in a variable upfront

Reply:
Changed and resolved. If unhappy with the provided solution, just unresolve :)